src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/AdditionalConditionPss.kt

```diff
@@ -10,4 +10,5 @@ data class AdditionalConditionPss(
   val pssDates: Boolean? = null,
   override val inputs: List<Input>? = null,
   override val type: String? = null,
+  override val skipable: Boolean? = null,
```

Comment:
is null a sensible default? making this mandatory might be better, you could default to false?

Reply:
Happy to default to false, I didn't want to bog-down the config with unnecessary declarations of `skippable = false` because that should be the assumption!
I _think_ there was a reason I went with `null` but I'll double check my own logic and can always refactor to account for default `false`